databox/api/src/Command/WorkspaceAsTemplateCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Entity\Core\Workspace;
+use App\Workspace\WorkspaceTemplater;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand('app:workspace:template')]
+class WorkspaceAsTemplateCommand extends Command
+{
+    public function __construct(
+        private readonly EntityManagerInterface $em,
+        private readonly WorkspaceTemplater     $workspaceTemplater,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this
+            ->setDescription('Export or import a workspace as a template')
+            ->addOption('export', null, InputOption::VALUE_REQUIRED, 'Workspace id to export')
+            ->addOption('import', null, InputOption::VALUE_REQUIRED, 'Workspace name to import into')
+            ->addOption('owner', null, InputOption::VALUE_REQUIRED, 'Owner id for of the imported workspace')
+            ->setHelp(<<<'HELP'
+Export a workspace as a template, or import a workspace from a template.
+
+- When only exporting:
```

Comment:
Please make 2 distinct commands `app:workspace:import` and `app:workspace:export`